### DISCUSSION TITLE
Configuring External Services via AppHost with Mocking Support

### DISCUSSION THREAD

**Discussion by chertby** [microsoft/aspire#5314] [Ideas]
What do you think about adding the ability to configure external services (accessible via REST APIs) through `AppHost`, similar to how internal projects are currently configured? Additionally, it would be helpful to allow these services to be swapped out with mocks, for example, using `WireMock.Net`.

**Comment by davidfowl** [comment]
It seems 2 people had this idea already https://www.nuget.org/packages?q=wiremock+aspire&includeComputedFrameworks=true&prerel=true&sortby=relevance 😄
